packages/react/src/components/ErrorSummary/ErrorSummaryHeading.tsx

```diff
@@ -1,31 +1,45 @@
-import { useContext, useEffect } from 'react';
+import { forwardRef, useContext, useEffect } from 'react';
 
-import type { ListHeadingProps } from '../List';
-import { List } from '../List';
+import { Heading, type HeadingProps } from '../Typography/Heading';
 
-import { ErrorSummaryContext } from './ErrorSummaryRoot';
+import {
+  ErrorSummaryContext,
+  type ErrorSummaryProps,
+} from './ErrorSummaryRoot';
 
-export type ErrorSummaryHeadingProps = ListHeadingProps;
+export type ErrorSummaryHeadingProps = Omit<HeadingProps, 'ref'>;
```

Comment:
Why did we need to omit `ref` here?

Reply:
Good catch, it is not really needed - just old habit of adding a `Omit<, 'ref'>` when the component adds its own new ref (through `forwardRef`) anyway, but I'll remove to keep consistent with other components.